### DISCUSSION TITLE
无法打开cGoogle Scholar

### DISCUSSION THREAD

**Discussion by Username-avaliable** [getlantern/lantern-forum-cn#434] [General]
can not open Google Scholar, showing Errorr 403 forbidden

**Comment by MaxMKMax** [comment]
@Username-avaliable 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 谢谢。

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Reply by Username-avaliable** [reply]
您好， 8 位数号码为71397573

**Reply by MaxMKMax** [reply]
@Username-avaliable 
好的谢谢您. 我会帮您提交问题给技术部门!

**Reply by MaxMKMax** [reply]
@Username-avaliable 您好. 我们的技术部门说您无法使用的原因是因为不是最新版本. 请您更新一下最新版本. 谢谢.

https://github.com/getlantern/lantern

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by Username-avaliable** [reply]
更新后问题仍然存在， 8 位数号码：71749361

**Reply by MaxMKMax** [reply]
@Username-avaliable 嗯嗯. 您确定是 8.0.0 版本对吧? 好的, 我马上帮您反馈.
